src/modules/Schedule/components/ColorPicker/constants.ts

```diff
@@ -0,0 +1,6 @@
+export const GENED_COLORS = ['pink', 'secondary', 'purple', 'green'] as const
+export const OTHER_COLORS = ['orange', 'teal', 'blue', 'indigo', 'primary'] as const
+export const SCHEDULE_COLORS = [...GENED_COLORS, ...OTHER_COLORS] as const
+
+export type GenEdColor = typeof GENED_COLORS[number]
```

Comment:
Where we use this?

Reply:
I don't remember anymore lol
So let me remove that